eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,49 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo ($serviceDirectoryName="*")
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try
+  {
+    pip install packaging==20.4 -q -I
+    $allSetupProps = (python -c "import sys; import os; sys.path.append(os.path.join('eng', 'scripts')); \
+    import get_package_properties; get_package_properties.get_all_package_properties('$serviceDirectoryName')")
+  }
+  catch
+  {
+    # This is soft error and failure is expected for python metapackages
+    LogError "Failed to get all package properties"
+  }
+  Pop-Location
+
+  foreach ($line in $allSetupProps)
+  {
+    $setupInfo = $line -Split ","
+    $packageName = $setupInfo[0].Trim("(' ")
```

Comment:
What cases would we have `(` that we would need to trim?

Reply:
The returned lines contain brackets. e.g. `('azure-common', '1.1.27', False, 'sdk/core\\azure-common')` So after spliting on `,` the brackets still need to be trimmed off